src/test/java/smile/identity/core/SignatureTest.java

```diff
@@ -0,0 +1,48 @@
+package smile.identity.core;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Date;
+
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.junit.Before;
+import org.junit.Test;
+
+public class SignatureTest {
+	
+    private Integer PARTNER_ID = 212;
```

Comment:
This is partner sensitive partner info here and should not be commited please remove from git history as this is a public repo

Reply:
Valid observation! 👍🏽